flang/test/Driver/lto-flags.f90

```diff
@@ -31,5 +31,6 @@
 ! THIN-LTO-ALL: "-fc1"
 ! THIN-LTO-ALL-SAME: "-flto=thin"
 ! THIN-LTO-LINKER-PLUGIN: "-plugin-opt=thinlto"
+! THIN-LTO-LINKER-PLUGIN-AIX: "-bdbg:thinlto"
```

Comment:
Is `-bdbg:thinlto` also related to a linker plugin? If not, it would be better to change the check prefix to something like `THIN-LTO-LINKER-FLAG-AIX`.